src/aatoolbox/utils/raster.py

```diff
@@ -471,7 +502,7 @@ class AatRasterDataset(AatRasterMixin, RasterDataset):
     def __init__(self, xarray_object):
         super().__init__(xarray_object)
 
-    def get_raster_array(self, var: str) -> xr.DataArray:
+    def get_raster_array(self, array: str) -> xr.DataArray:
```

Comment:
This name is a bit confusing because it's a string type called `array`. Maybe call it `variable_name`?